azure-pipelines.yml

```diff
@@ -1,31 +1,25 @@
 jobs:
 - job: linux
   pool: {vmImage: 'Ubuntu-16.04'}
-  steps: 
+  steps:
     - task: UsePythonVersion@0
     - bash: |
         python -m pip install -r requirements-dev.txt
         python ./bin/run_tests.py
 
 - job: macos
   pool: {vmImage: 'macOS-10.13'}
-  steps: 
+  steps:
     - task: UsePythonVersion@0
     - bash: |
         python -m pip install -r requirements-dev.txt
         python ./bin/run_tests.py
 
 - job: windows
   pool: {vmImage: 'vs2017-win2016'}
-  steps: 
+  steps:
     - {task: UsePythonVersion@0, inputs: {versionSpec: '2.7', architecture: x86}}
     - {task: UsePythonVersion@0, inputs: {versionSpec: '2.7', architecture: x64}}
-    - {task: UsePythonVersion@0, inputs: {versionSpec: '3.5', architecture: x86}}
-    - {task: UsePythonVersion@0, inputs: {versionSpec: '3.5', architecture: x64}}
-    - {task: UsePythonVersion@0, inputs: {versionSpec: '3.6', architecture: x86}}
-    - {task: UsePythonVersion@0, inputs: {versionSpec: '3.6', architecture: x64}}
-    - {task: UsePythonVersion@0, inputs: {versionSpec: '3.7', architecture: x86}}
-    - {task: UsePythonVersion@0, inputs: {versionSpec: '3.7', architecture: x64}}
```

Comment:
Hmmm, so this is now part of testing the new mechanism to install?
Maybe we could/should:
1. Add another job that tests the 2.7 installation? (Just setting `CIBW_BUILD` to `cp27` or so?)
2. Document that these versions are not listed so that the new "install-if-not-present" mechanism is tested?
3. (or 2.bis) Make sure it is clear/documented to the `cibuildwheel` users that want to use this file as an example configuration that it is recommended to not do it this way, but rather still add these versions, as it is faster?

Reply:
It's the only recommended way if you wan't python 2.7 support. Other python versions requires the installer to be run on Azure.

Reply:
Aha, yes, because we don't know the origin of the Python versions on 2.7?

In that case, the example configuration in `README` should be adapted to reflect this change, though.

Reply:
The `README` is in sync with this change (or am I missing another line in the `README` where it's used ?)

Reply:
Apologies, I'd overlooked that! :-/